rustup-init.ps1

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env powershell
```

Comment:
PowerShell Core is not available here.  `/usr/bin/pwsh` is the correct location
https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-linux?view=powershell-6#uninstalling-binary-archives

Reply:
Ah, that makes sense (I took this from another script) and it makes sense who consumes this (i.e. non-Windows systems).